From 2d12c4c5c33b577ba24c7d20519448fc6d1cb0c4 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 26 Sep 2013 17:57:18 +0800
Subject: [PATCH 0196/1074] ENGR00281446 ARM: dts: imx6: correct the power
 supply node name for epdc

The naming convention for power supply node in DTS is "%s-supply".
With this patch regulator_get() will process in the DT way rather than
traditional way. This patch has no functional impact.

Signed-off-by: Robby Cai <R63905@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6dl-sabresd.dts |    6 +++---
 arch/arm/boot/dts/imx6sl-evk.dts     |    6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabresd.dts b/arch/arm/boot/dts/imx6dl-sabresd.dts
index 52f0625..e1d22e7 100644
--- a/arch/arm/boot/dts/imx6dl-sabresd.dts
+++ b/arch/arm/boot/dts/imx6dl-sabresd.dts
@@ -19,9 +19,9 @@
 &epdc {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_epdc_0>;
-	V3P3_supply = <&V3P3_reg>;
-	VCOM_supply = <&VCOM_reg>;
-	DISPLAY_supply = <&DISPLAY_reg>;
+	V3P3-supply = <&V3P3_reg>;
+	VCOM-supply = <&VCOM_reg>;
+	DISPLAY-supply = <&DISPLAY_reg>;
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 8a8ec72..f3f967b 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -174,9 +174,9 @@
 &epdc {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_epdc_0>;
-	V3P3_supply = <&V3P3_reg>;
-	VCOM_supply = <&VCOM_reg>;
-	DISPLAY_supply = <&DISPLAY_reg>;
+	V3P3-supply = <&V3P3_reg>;
+	VCOM-supply = <&VCOM_reg>;
+	DISPLAY-supply = <&DISPLAY_reg>;
 	status = "okay";
 };
 
-- 
1.7.5.4

